includes/find.form.inc

```diff
@@ -158,6 +163,7 @@ function islandora_find_replace_find_form_submit($form, &$form_state) {
 
   $find_replace = array(
     'cmodel' => $form_state['values']['content_model'],
+    'namespace' => $form_state['values']['namespace'],
```

Comment:
This can use a trim()

Reply:
How/why would trim() apply here?

Reply:
Just to avoid spaces after/before namespace in the input. The the search function is pretty expensive and I felt if we clean namespace from at least extra spaces we can avoid a non sense search. But not enforced == so many other ways to make it fail!. Lovely would be to have a form validation really in the future. (Dismiss if you feel you can't do it now)

Reply:
OK, I'm just not sure how I would add that here. 

`'namespace' => trim($form_state['values']['namepace'])` ?

Reply:
Yes, Simplest way for now. I will go for a full-blown validation handler during the weekend.